.pfnci/config.pbtxt

```diff
@@ -24,7 +24,7 @@ configs {
     time_limit {
       seconds: 10800
     }
-    command: ".pfnci/wheel-linux/main.sh rocm-5.0 3.9"
+    command: ".pfnci/wheel-linux/main.sh rocm-7.0 3.12"
```

Comment:
Does this mean the old ROCm 7.0 (latest: 7.1.1) isn't compatible with Python 3.14 on Linux yet, or is it something like a minimum?

Reply:
No, this is just a CI configuration that intends to test one of our support matrix. We'll ship wheels in full matrix we support (Python 3.10-3.14 x CUDA 12 / CUDA 13 / ROCm 7.0).

Reply:
We have been pushing wheels for many years. I don't understand what you meant by "publishing wheels for the first time." Please read https://docs.cupy.dev/en/stable/install.html#installing-cupy.